nanome/_internal/_structure/_serialization/_atom_serializer.py

```diff
@@ -91,6 +92,9 @@ def serialize(self, version, value, context):
         if version >= 7:
             context.write_uint(value._display_mode)
 
+        if version >= 8:
+            context.write_float(value._vdw_radius)
```

Comment:
I'm wondering if it makes sense to add van der wall radius to serialization and have that information sent by nanome and back, and have it not used by nanome.
Since this information is constant, I think we should preferably have an element table like in Nanome, and fetch the vdw radius from there when calling vdw_radius getter

Reply:
Please check my implementation of the atom table in _atom.py, I tried to use a classmethod but we can probably find a better solution to fill the dictionary.